src/main/java/pages/yandex/MailPage.java

```diff
@@ -12,6 +12,7 @@
     private static final By WRITE_EMAIL = By.xpath("//i[contains(text(),'Compose')]");
```

Comment:
Line 11: not existing element? Remove it

Reply:
?

Reply:
YandexLogo locators removed